backend/api/users/views.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render, redirect  # noqa
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.settings import api_settings
+
+from api.users.serializers import AuthTokenSerializer
+
+
+class LoginEndpoint(ObtainAuthToken):
```

Comment:
It looks like the `ObtainAuthToken` already handles Token creation, you only need to ensure there's a `user` in the `validated_data` of the serializer: https://github.com/encode/django-rest-framework/blob/master/rest_framework/authtoken/views.py#L54

Reply:
Awesome!